[TASK] Add a stub for TYPO3 Administration (#4844)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/Administration/Index.rst

:navigation-title: Administration
..  include:: /Includes.rst.txt
..  _administration:

====================
TYPO3 administration
====================

..  card-grid::
    :columns: 1
    :columns-md: 2
    :gap: 4
    :card-height: 100

    ..  card:: TYPO3 Installation (Composer)

>         The getting started guides covers each of the steps required to

Reviewer: ```suggestion
        The getting started guide covers every step required to
```

        install TYPO3 using Composer.

        :ref:`Getting started: TYPO3 installation <t3start:install>`

    ..  card:: Upgrading TYPO3

>         The TYPO3 upgrade guide explains how to do patch level update, how

Reviewer: ```suggestion
        The TYPO3 upgrade guide explains how to do patch level updates and how
```
